app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name), class:'log-in-shared' %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name), class:'sign-up-shared' %><br />
+<% end %>
+
+<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+  <%= link_to "Forgot your password?", new_password_path(resource_name), class:'forget-password-shared' %><br />
+<% end %>
+
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
+  <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name), class: 'confirmations-shared' %><br />
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
+  <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), data: { turbo: false } %><br />
+  <% end %>
```

Comment:
- [ ] It would be nice if you could modify your styles for these links to show in one line. Currently, your styles are meant for desktop view, and in desktop view everything shows fine. But according to the project requirement, the app is a web mobile app, and therefore, your styles must match a mobile app's looks. Kindly make these links display in one line 👍🏽

<img width="455" alt="Screenshot 2023-06-03 at 1 41 44 PM" src="https://github.com/The-Normal-One1/budget_app/assets/50941074/1cad388c-cbdc-47a0-8e9f-ac52bddc6a3c">